tests/e2e/utils/payments.js

```diff
@@ -300,14 +300,19 @@ export async function setupCheckout( billingDetails ) {
 /**
  * Selects the Giropay payment method on the checkout page.
  *
+ * @param {*} paymentMethod The payment method to select.
  * @param {*} page The page reference object.
  */
-export async function selectGiropayOnCheckout( page ) {
-	await page.$( '#payment .payment_method_woocommerce_payments_giropay' );
-	const giropayRadioLabel = await page.waitForSelector(
-		'#payment .payment_method_woocommerce_payments_giropay label'
+export async function selectOnCheckout( paymentMethod, page ) {
```

Comment:
This is really nice refactor and makes this util much more useful! The description could be updated now that it can handle multiple payment methods.

Reply:
Thank you! 809d05dbb87875648f2df0c45a47d6ee77511c3a